docs/nva-publication-channels.yaml

```diff
@@ -22,6 +22,62 @@ externalDocs:
 servers:
   - url: https://api.nva.unit.no/channels
 paths:
+  /journal:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+            Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${PublicationChannelsHandler.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+      - Journal
+      summary: Find journal by ID
+      description: Returns a single Journal
+      operationId: GetJournalById
+      parameters:
+      - name: query
+        in: query
+        description: Term to search for in publication
+        required: true
+        schema:
+          type: string
+        example: '&journal%'
```

Comment:
Not sure what this is… if it is your intention to provide a journal name, 'Journal of pygmy sheep' might be a better example

Reply:
It's a fat-fingered typo, should have been '%journal%' - but changed to 'Journal of pygmy sheep'